### DISCUSSION TITLE
How do I set `with_max_level` ?

### DISCUSSION THREAD

**Discussion by rain2307** [tokio-rs/tracing#2695] [Q&A] [answered]
https://gist.github.com/rain2307/82c0129196ed76f0f1b28d1a5534d1a4

I have tried to set `.with_max_level` on `fmt::layer()` but it is not available.

I'm trying to output to the terminal and log file at the same time.

**Comment by davidbarsky** [comment, marked as answer]
Sorry for the delay. That is a helper method on `SubscriberBuilder` that we should probably mark as deprecated. Since `LevelFilter` [implements the `Filter` trait](https://docs.rs/tracing-subscriber/latest/tracing_subscriber/filter/struct.LevelFilter.html#impl-Filter%3CS%3E-for-LevelFilter), you can filter levels for individual layers using filters, as described in [the documentation here](https://docs.rs/tracing-subscriber/latest/tracing_subscriber/layer/index.html#filtering-with-layers).

**Reply by rain2307** [reply]
Thank you very much. Finally, it's working properly.
It would be great to have more examples to learn from.
I have updated the Gist. Corrections and feedback are welcome.
